source/Octopus.Client/Repositories/LicensesRepository.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using Octopus.Client.Editors;
+using Octopus.Client.Model;
+
+namespace Octopus.Client.Repositories
+{
+    public interface ILicensesRepository
+    {
+        LicenseResource GetCurrent();
+        LicenseResource UpdateCurrent(LicenseResource resource);
+        LicenseStatusResource GetStatus();
+    }
+    
+    class LicensesRepository : BasicRepository<LicenseResource>, ILicensesRepository
+    {
+        public LicensesRepository(IOctopusRepository repository)
+            : base(repository, "CurrentLicense")
+        {
+        }
+
+        public LicenseResource GetCurrent()
+            => Client.Get<LicenseResource>(Repository.Link(CollectionLinkName));
+
+        public LicenseResource UpdateCurrent(LicenseResource resource)
```

Comment:
`BasicRepository` has `Modify` method which does what `UpdateCurrent` does here.  Should we use instead ?

Reply:
I did consider that, however I decided to go this direction so that it compliments `GetCurrent`. Also if we ever add updating arbitrary licences (probably never), the `Update` method would still be available.